src/Packagist/WebBundle/Resources/views/Package/viewPackage.html.twig

```diff
@@ -12,6 +12,20 @@
 
 {% block scripts %}
     <script src="{{ asset('bundles/packagistweb/js/view.js')}}"></script>
+    
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.8.0/styles/monokai_sublime.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.8.0/highlight.min.js"></script>
+    <script>
+      $(document).ready(function() {
+        hljs.configure({
+          tabReplace: 4,
+          languages: ['apache', 'bash', 'ini', 'javascript', 'json', 'nginx', 'php', 'sql', 'css', 'html']
```

Comment:
Maybe asynchronously highlight jobs would be a solution for no languages limit.

See: https://github.com/isagalaev/highlight.js/issues/954#issuecomment-150802447